templates/design_doc/src/lib/actions.coffee

```diff
@@ -1,5 +1,5 @@
-do_action = require('./shared/do_action')
-validate_doc_update = require('./shared/validate_doc_update').validate_doc_update
+do_action = require('./pantheon-helpers').design_docs.do_action
+validate_doc_update = require('pantheon-helpers').design_docs.validate_doc_update
```

Comment:
Are you sure this works? To me it looks like [design_docs.validate_doc_update](https://github.com/dgreisen-cfpb/pantheon-helpers/blob/master/lib/pantheon-helpers-design-docs/validate_doc_update.js) is an object that contains the function `validate_doc_update`, meaning this should be:

```
validate_doc_update = require('pantheon-helpers').design_docs.validate_doc_update.validate_doc_update
```